src/components/Randombeer.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useState, useEffect } from 'react';
+import Header from './Header';
+import axios from 'axios';
+
+function Randombeer() {
+  const [random, setRandom] = useState({});
+  const [randomBeerDetail, setRandomBeerDetail] = useState({});
+
+  const getFromApi = async () => {
+    try {
+      const apiCall = await axios.get(`https://ih-beers-api2.herokuapp.com/beers/${random}`);
+      setRandom(apiCall.data); 
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+
+   getFromApi()
+
+  }, [])
+  return (
+    <div>
+        <Header/>
+          <img src={randomBeerDetail.image_url} alt=""/>
```

Comment:
And since you're using `setRandom` on your API call, you should use `random` to display the beers here, like:

`<h2>{random.name}</h2>`